roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -178,14 +171,14 @@
     "identity:delete_protocol": "rule:admin_required",
 
     "identity:create_mapping": "rule:admin_required",
-    "identity:get_mapping": "rule:admin_required",
-    "identity:list_mappings": "rule:admin_required",
+    "identity:get_mapping": "rule:admin_or_cloudadmin",
+    "identity:list_mappings": "rule:admin_or_cloudadmin",
     "identity:delete_mapping": "rule:admin_required",
-    "identity:update_mapping": "rule:admin_required",
+    "identity:update_mapping": "rule:admin_or_cloudadmin",
```

Comment:
No they cannot. Jellyroll RBAC filter should not allow them to grant the admin role to any group. This will let them update the mappings so that they are able to map the usergroups to these new  projects. If we do not provide this capability, the operations team will have to do this for them and it could become very complicated to manage.